hadoop-hdds/rocksdb-checkpoint-differ/src/main/java/org/apache/ozone/rocksdb/util/RdbUtil.java

```diff
@@ -49,4 +55,17 @@ public static Set<String> getSSTFilesForComparison(
         .map(lfm -> new File(lfm.path(), lfm.fileName()).getPath())
         .collect(Collectors.toCollection(HashSet::new));
   }
+
+  public static Map<Object, String> getSSTFilesWithInodesForComparison(final ManagedRocksDB rocksDB, List<String> cfs) {
+    return getLiveSSTFilesForCFs(rocksDB, cfs).stream()
+        .map(lfm -> {
+          File sstFile = new File(lfm.path(), lfm.fileName());
+          try {
+            Object inode = Files.readAttributes(sstFile.toPath(), BasicFileAttributes.class).fileKey();
+            return new AbstractMap.SimpleEntry<>(inode, sstFile.getPath());
+          } catch (IOException e) {
+            throw new UncheckedIOException("Failed to read inode for " + sstFile.getPath(), e);
```

Comment:
Don't throw UncheckedIOException throw the actual IOException instead.

Reply:
lambda doesnt allow throwing a checked (Actual IOException)

Reply:
Don't use lambda or catch the uncheckedIOException and throw e.getCause()